src/pcsswebservice/src/main/resources/wsdl/JusticeCDDS.wsdl

```diff
@@ -0,0 +1,135 @@
+          <xsd:simpleType name="AgencyIdentifierType">
+            <xsd:restriction base="xsd:string">
+              <xsd:whiteSpace value="preserve"/>
+              <xsd:maxLength value="16"/>
+            </xsd:restriction>
+          </xsd:simpleType>
+          <xsd:simpleType name="SystemIdType">
+            <xsd:restriction base="xsd:string">
+              <xsd:whiteSpace value="preserve"/>
+              <xsd:maxLength value="16"/>
+            </xsd:restriction>
+          </xsd:simpleType>
+          <xsd:simpleType name="DateType">
+            <xsd:restriction base="xsd:string">
+              <xsd:whiteSpace value="preserve"/>
+              <xsd:length value="21"/>
+            </xsd:restriction>
+          </xsd:simpleType>
+          <xsd:simpleType name="CourtRoomCdType">
+            <xsd:restriction base="xsd:string">
+              <xsd:whiteSpace value="preserve"/>
+              <xsd:maxLength value="6"/>
+            </xsd:restriction>
+          </xsd:simpleType>
+          <xsd:simpleType name="TimeType">
+            <xsd:restriction base="xsd:string">
+              <xsd:whiteSpace value="preserve"/>
+              <xsd:length value="21"/>
+            </xsd:restriction>
+          </xsd:simpleType>
+        	</xsd:schema>
+  </wsdl:types>
+  <wsdl:message name="cdds_PortType_getDigitalDisplayCourtListResponse">
+    <wsdl:part name="parameters" element="tns:getDigitalDisplayCourtListResponse">
+    </wsdl:part>
+  </wsdl:message>
+  <wsdl:message name="cdds_PortType_getDigitalDisplayCourtList">
+    <wsdl:part name="parameters" element="tns:getDigitalDisplayCourtList">
+    </wsdl:part>
+  </wsdl:message>
+  <wsdl:portType name="cdds_PortType">
+    <wsdl:operation name="getDigitalDisplayCourtList">
+      <wsdl:input message="tns:cdds_PortType_getDigitalDisplayCourtList">
+    </wsdl:input>
+      <wsdl:output message="tns:cdds_PortType_getDigitalDisplayCourtListResponse">
+    </wsdl:output>
+    </wsdl:operation>
+  </wsdl:portType>
+  <wsdl:binding name="JusticeCDDS_wsProvider_cdds_Binder" type="tns:cdds_PortType">
+    <soap12:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+    <wsdl:operation name="getDigitalDisplayCourtList">
+      <soap12:operation soapAction="JusticeCDDS_wsProvider_cdds_Binder_getDigitalDisplayCourtList" style="document"/>
+      <wsdl:input>
+        <soap12:body parts="parameters" use="literal"/>
+      </wsdl:input>
+      <wsdl:output>
+        <soap12:body parts="parameters" use="literal"/>
+      </wsdl:output>
+    </wsdl:operation>
+  </wsdl:binding>
+  <wsdl:service name="JusticeCDDS.wsProvider.cdds">
+    <wsdl:port name="JusticeCDDS_wsProvider_cdds_Port" binding="tns:JusticeCDDS_wsProvider_cdds_Binder">
+      <soap12:address location="http://reeks.bcgov:5556/ws/JusticeCDDS.wsProvider.cdds/JusticeCDDS_wsProvider_cdds_Port"/>
+    </wsdl:port>
+  </wsdl:service>
+</wsdl:definitions>
```

Comment:
<3

Reply:
We are keeping this to be compatible with webMethods sucks but that's what it is...

Reply:
I was mentioning the lack of missing newlines.